### ISSUE TITLE
Error in Drop: Failed to remove user cgroup

### ISSUE THREAD

**Issue by vanutp** [purplesyringa/sunwalker-box#7]
In some situations sunwalker-box `run` command fails with the following error message:
```
Error in Drop: Failed to remove user cgroup

Caused by:
    0: Failed to rmdir "box-GQepE22LOZ"
    1: EBUSY: Device or resource busy
error "Unknown pid 6"
```
The sandbox is unusable afterwards, i.e. `reset` command fails.

Steps to reproduce:
1. Create rootfs using [this](https://gist.github.com/vanutp/8a6961932d92cf42d18a65d1100d2cc8) Dockerfile
2. Run sunwalker-box with `--root /path/to/rootfs`
3. Run commands:
```
run {"argv": ["/usr/bin/cp", "/meow.cs", "/meow.csproj", "."]}
run {"argv": ["/usr/bin/bash", "-c", "dotnet restore --source /nuget_packages && dotnet publish --no-restore"], "env": {"DOTNET_CLI_TELEMETRY_OPTOUT": "1", "DOTNET_NOLOGO": "1", "HOME": "/space"}}
```
---
If the second command is split to
```
run {"argv": ["/usr/bin/bash", "-c", "dotnet restore --source /nuget_packages"], "env": {"DOTNET_CLI_TELEMETRY_OPTOUT": "1", "DOTNET_NOLOGO": "1", "HOME": "/space"}}
run {"argv": ["/usr/bin/bash", "-c", "dotnet publish --no-restore"], "env": {"DOTNET_CLI_TELEMETRY_OPTOUT": "1", "DOTNET_NOLOGO": "1", "HOME": "/space"}}
```
first command works properly, but second returns Signaled, exit_code: -7

**Comment by yuki0iq** [comment]
Error in Drop should not happen on master now, starting from https://github.com/purplesyringa/sunwalker-box/commit/94a43986648e3252856d742b56f7f50fe70a72ba

**Comment by purplesyringa** [comment]
A more permanent fix has landed on dev in 49f10fc5db9a2360f8350944cae5476623095879. Please check if it solves your problem!

**Comment by vanutp** [comment]
Works perfectly, thanks!
